template.yaml

```diff
@@ -1389,6 +1320,29 @@ Resources:
   # Secrets
   #
 
+  # The value of this secret is a json blob containing the configuration for all RP stubs in this environment.
+  # Each stub instance uses the same configuration but with a different default client ID.
+  RpConfigurationSecret:
+    Type: AWS::SecretsManager::Secret
+    Properties:
+      Name: !Sub /stubs/rp-${Environment}/CONFIGURATION
+      KmsKeyId: !GetAtt KmsKey.Arn
+
+  ConfigurationSecretsManagerAccessPolicy:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Action:
+              - secretsmanager:GetSecretValue
+            Resource: !Ref RpConfigurationSecret
+          - Effect: Allow
+            Action: kms:Decrypt
+            Resource: !GetAtt KmsKey.Arn
+
+  # BEGIN remove
```

Comment:
I'll remove these in a followup, just leaving them here for now to make rollback easier